### PR TITLE
[2.0 api] fix the example codes in doc strings

### PR DESCRIPTION
**Pull request by sandyhouse**
### PR types

Others
### PR changes

Docs
### Describe

fix the example codes in doc strings

### CODE REVIEW

File: python/paddle/tensor/manipulation.py [hunk 1]
```diff
@@ -946,16 +952,16 @@ def expand(x, shape, name=None):
...
             # [[1, 2, 3], [1, 2, 3]]
 
-            np_shape = np.array([2, 3]).astype=('int32)
-            shape = paddle.to_variable(np_shape)
+            np_shape = np.array([2, 3]).astype('int32')
+            shape = paddle.to_tensor(np_shape)
             out = paddle.expand(data, shape=shape)
-			out = out.numpy
+			out = out.numpy()
             # [[1, 2, 3], [1, 2, 3]]
```
Comment (TCChenlong) [inline python/paddle/tensor/manipulation.py:965]: 这里的格式对齐哈~

**Review by TCChenlong — APPROVED** [review]
LGTM

File: python/paddle/tensor/manipulation.py (hunk 1 above)
Comment (sandyhouse) [reply python/paddle/tensor/manipulation.py:965]: A new pr to fix it.

**Review by gavin1332 — APPROVED** [review]
LGTM

**Pull request merged**